packages/components/src/internal/components/domainproperties/PropDescType.ts

```diff
@@ -75,6 +76,10 @@ export class PropDescType
         return conceptURI === STORAGE_UNIQUE_ID_CONCEPT_URI;
     }
 
+    static isTextChoice(name: string): boolean {
+        return name === TEXT_CHOICE_CONCEPT_URI || name === 'textChoice';
```

Comment:
For the field row data type select input on change event, looks like it passes the field input name prop instead of the concept URI here (so that it can apply for non-conceptURI based data types).

Reply:
Is it worth adding a comment about that?

Reply:
Done